spec/restaurant_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'restaurant'
+
+describe Restaurant do
+
+  it 'has a menu of dishes with prices' do
+    expect(subject.menu).to eq({margherita: 5, pepperoni: 6, hawaiian: 6, supreme: 7})
```

Comment:
Line is too long. [86/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.